src/network/networkd-route.c

```diff
@@ -2631,7 +2631,7 @@ int config_parse_tcp_advmss(
         return 0;
 }
 
-int config_parse_tcp_window(
+int config_parse_route_dhcp_tcp_window(
```

Comment:
This is generic and not specific for dhcp. So, please rename this to `config_parse_tcp_window()`.